Pages/Product-Page/product.html

```diff
@@ -0,0 +1,433 @@
+                            </button>
+                            <button class="card-icon-btn outline-card-secondary-btn"><i class="fa fa-heart-o fa-x" aria-hidden="true"></i></button>
+                        </div>
+                        <div class="badge-on-card">
+                            Popular
+                        </div>
+                        <button class="card-dismiss-btn outline-card-secondary-btn">X</button>
+                    </div>
+                </div>
+
+                <div class="card-basic">
+                    <img src="../../Images/Product-Page/Naruto Volume-1.jpg" alt="Manga-Naruto Volume-1">
+                    <div class="card-item-details">
+                        <h3>Naruto Manga Volume-1</h3>
+                        <h4>- By Masashi Kishimoto</h4>
+                        <p><b>&#8377; 300 &nbsp;&nbsp;</b><del>&#8377; 400</del> &nbsp;&nbsp;
+                            <p class="discount-on-card">25% off</p>
+                        </p>
+                        <div class="card-button">
+                            <button class="outline-card-secondary-btn">
+                                <i class="fa fa-shopping-cart fa-x" aria-hidden="true" ></i>
+                                &nbsp;Add To Cart
+                            </button>
+                            <button class="card-icon-btn outline-card-secondary-btn"><i class="fa fa-heart-o fa-x" aria-hidden="true"></i></button>
+                        </div>
+                        <div class="badge-on-card">
+                            Manga
+                        </div>
+                        <button class="card-dismiss-btn outline-card-secondary-btn">X</button>
+                    </div>
+                </div>
+
+                <div class="card-basic">
+                    <img src="../../Images/Product-Page/Naruto Volume-72.jpg" alt="Manga-Naruto Volume-72">
+                    <div class="card-item-details">
+                        <h3>Naruto Manga Volume-1</h3>
+                        <h4>- By Masashi Kishimoto</h4>
+                        <p><b>&#8377; 300 &nbsp;&nbsp;</b><del>&#8377; 400</del> &nbsp;&nbsp;
+                            <p class="discount-on-card">25% off</p>
+                        </p>
+                        <div class="card-button">
+                            <button class="outline-card-secondary-btn">
+                                <i class="fa fa-shopping-cart fa-x" aria-hidden="true" ></i>
+                                &nbsp;Add To Cart
+                            </button>
+                            <button class="card-icon-btn outline-card-secondary-btn"><i class="fa fa-heart-o fa-x" aria-hidden="true"></i></button>
+                        </div>
+                        <div class="badge-on-card">
+                            Manga
+                        </div>
+                        <button class="card-dismiss-btn outline-card-secondary-btn">X</button>
```

Comment:
In the dismiss button there is some margin to the right side but no margin from the top side, margin on all 4 sides would make it more consistent

Reply:
Good point, plus by mistake I added cards with dismiss but cards with dismiss are not needed on product page, so I will remove those buttons.